cmd/amplifier/server/configuration.go

```diff
@@ -10,17 +10,17 @@ import (
 )
 
 const (
-	DefaultPort          = ":50101"
-	DefaultPublicAddress = "local.appcelerator.io"
-	DefaultTimeout       = time.Minute
+	DefaultPort       = ":50101"
+	DefaultPublicHost = "127.0.0.1"
```

Comment:
I can add an environment variable for port as well, or group both address and port into a single value, such as `PublicEndpoint`. Tell me what you think is best.